aries-site/src/pages/components/layer.mdx

```diff
@@ -71,8 +73,7 @@ Provides users with any type of action, required or optional. These can be confi
 
 Actionable Layers contain the following elements and interactions: 
 - A cancel or close button will always be positioned within the footer of the Layer. Note that the "X" icon is omitted to avoid any confusion the user may have on which option they should use to close out the Layer. 
-- The Layer heading and action buttons should be sticky while the rest of the Layer content is scrollable.
-Provides users with any type of action, required or optional. These can be confirmations, filters, forms, etc.
+- For situations where the contents exceed the height of the layer, the header should either always stick atop the Layer while the content scrolls beneath, or to maximize screen real-estate, the header may scroll away as the user scrolls down the content, but re-appear as the user scrolls up, signaling their intent to return to the top of the content.
```

Comment:
Can we link to each of the examples in this?
```suggestion
- The Layer header should be sticky while the rest of the Layer content is scrollable. This can either be achieved with a [persistent sticky header]() or a [header that is sticky when the user scrolls up](). 
```